test/runtests.jl

```diff
@@ -19,6 +19,6 @@ FT = Float64
 
 saved_values = SavedValues(FT, ClimaCore.Fields.FieldVector)
 
-include("./initial_structure_test.jl")
+#include("./initial_structure_test.jl")
```

Comment:
worth holding onto this test?

Reply:
Hm, this test was just making sure that the variables are stored correctly in Y and p when we have a landmodel, i think. I think it is fine to let this one go as we will have to more carefully write unit tests for all of clima lsm soon, and this structure set up is kind of implicitly tested in the lsm simulation test we do.